cms/templates/settings.html

```diff
@@ -291,6 +292,27 @@ <h2 class="title-2">${_('Course Schedule')}</h2>
               </div>
             </li>
           </ol>
+
+          % if upgrade_deadline:
+          <ol class="list-input">
```

Comment:
Nit: Not sure what convention is, but I'd prefer if this block was indented so it looks more like an if statement

Reply:
Rest of file didn't indent. :man_shrugging: